src/Summary.jsx

```diff
@@ -34,26 +34,27 @@ class Summary extends React.Component {
     };
   }
 
-  componentDidMount() {
+  async componentDidMount() {
     const { activeArea } = this.props;
-    const blockId = (activeArea && activeArea.id) ? activeArea.id : null;
-    const request = RestAPI.requestGeometryByArea(blockId);
-    this.setState({
-      layers: {
-        dagma: {
-          displayName: 'dagma',
-          id: 1,
-          active: true,
-          layer: L.geoJSON(request, {
-            style: {
-              stroke: false,
-              fillColor: '#5f8f2c',
-              fillOpacity: 0.7,
-            },
-          }),
+    if (activeArea) {
```

Comment:
also need to check if activeArea.name exists, it could be undefined or null. And if it doesn't exists you should report the connection error.

Check this, it's clearer: https://github.com/PEM-Humboldt/mibloque/blob/226ba7193cb87bf2c8ec5fe66a0c5c1e34aaef78/src/Indicator.jsx#L29